projects/js-packages/components/components/a8c-svg-title/README.md

```diff
@@ -0,0 +1,16 @@
+# A8C SVG Title
+
+Component that renders the stylish `AN AUTOMATTIC AIRLINE`.
+It takes width and height properties but defaults to 7 in height.
+
+#### How to use:
+
+```js
+<A8cSvgTitle height={ 7 } className="jp-a8c-svg-title" />
+```
+
+#### Props
+
+- `className`: String - (default: `jp-a8c-svg-title`) the class name set on the SVG element.
```

Comment:
Instead of overriding `.jp-a8c-svg-title`, might I recommend that we use the `classNames` module to allow developers to add to the existing class name like we do for the footer component below? Users of component libraries typically expect to be able to append class names onto components without losing the built-in styling.

Reply:
addressed in 4b25a83